profiles/common/modules/features/nexteuropa_newsroom/classes/NewsroomAgenda.inc

```diff
@@ -0,0 +1,679 @@
+<?php
+
+/**
+ * @file
+ * Code for the NewsroomAgenda class.
+ */
+
+/**
+ * NewsroomAgenda class base class.
+ */
+abstract class NewsroomAgenda {
+
+  const AGENDA_VIEW_SCHEDULE = 'schedule';
+  const AGENDA_VIEW_MONTH = 'month';
+  const AGENDA_VIEW_WEEK = 'week';
+  const AGENDA_VIEW_DAY = 'day';
+  const AGENDA_VIEW_SCHEDULE_BLOCK = 'schedule-block';
+
+  protected $view = NewsroomAgenda::AGENDA_VIEW_SCHEDULE;
+  protected $topicUrl = 'all';
+  protected $topics = array();
+  protected $typeUrl = 'all';
+  protected $typeId = NULL;
+  protected $currentDate = NULL;
+  protected $nextDate = NULL;
+  protected $previousDate = NULL;
+  protected $currentItems = array();
+  protected $nextItems = array();
+  protected $result = array();
+  protected $query = NULL;
+  protected $itemsCount = 0;
+  protected $urlParts = array();
+  protected $displayCalendar = TRUE;
+  protected $isBlock = FALSE;
+
+  /**
```

Comment:
i was refering here for the phpdoc documentations what are not written fully there. and right now the issue is still existing.